Extension/lib/filter/update-service.js

```diff
@@ -251,6 +261,28 @@ exports.ApplicationUpdateService = {
 	},
 
 	/**
+	 * Update Firefox storage by moving to prefs
+	 *
+	 * Version 2.1.1
+	 * @returns {exports.Promise}
+	 * @private
+	 */
+	_onUpdateFirefoxStorage: function () {
+		Log.info('Call update to version 2.1.1');
+
+		var dfd = new Promise();
+
+		var items = LS.items;
+		for (var i = 0; i < items.length; i++) {
```

Comment:
You can use for in ... loop for SimpleStorage.storage. It's simpler and you transfer only existing properties.